trunk/SUAVE/Components/Energy/Networks/Internal_Combustion_Propeller.py

```diff
@@ -0,0 +1,153 @@
+## @ingroup Components-Energy-Networks
+# Internal_Combustion_Propeller.py
+# 
+# Created:  Sep 2016, E. Botero
+# Modified: Apr 2018, M. Clarke 
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import SUAVE
+
+# package imports
+import numpy as np
+from SUAVE.Components.Propulsors.Propulsor import Propulsor
+from SUAVE.Core import Data, Units
+
+# ----------------------------------------------------------------------
+#  Network
+# ----------------------------------------------------------------------
+## @ingroup Components-Energy-Networks
+class Internal_Combustion_Propeller(Propulsor):
+    """ A simple mock up of an internal combustion propeller engine. Tis network adds an extra
```

Comment:
So the difference between propeller and this is the unknowns?  Is there a better way to reduce both files into one? (thinking about user friendliness)